doc/dev/crimson.rst

```diff
@@ -0,0 +1,143 @@
+=======
+crimson
+=======
+
+crimson is the code name of crimson-osd, which the next generation ceph-osd. It
+targets fast storage devices, like NVMe storage, for better performance.
+
+
+daemonize
+---------
+
+Unlike ``ceph-osd``, ``crimson-osd`` does daemonize itself even if the
+``daemonize`` option is enabled. Because, to read this option, ``crimson-osd``
+needs to ready its config sharded service, but this sharded service lives
```

Comment:
ready or read?

Reply:
see https://www.merriam-webster.com/dictionary/ready#h2-anchor

actually both should work, but if "read" is used, probably i would go with "read from".

Reply:
Ah, sorry. Indeed.